ARCore/app/src/main/java/com/novoda/spikes/arcore/helper/ARCoreDependenciesHelper.kt

```diff
@@ -0,0 +1,37 @@
+package com.novoda.spikes.arcore.helper
+
+import android.app.Activity
+import com.google.ar.core.ArCoreApk
+import com.google.ar.core.exceptions.*
+
+object ARCoreDependenciesHelper {
+    class Result(val isARCoreInstalled: Boolean, val message: String)
```

Comment:
This is a great case for a `sealed class` 😋 

```
sealed class Result {
  class Installed : Result
  class NotInstalled : Result
}
```

Reply:
Updated here 27b86a31